### ISSUE TITLE
Relative import plugin broken for Windows and when bundle/minification optimizations are added

### ISSUE THREAD

**Issue by water-a** [karolis-sh/electron-snowpack#58]
Firstly, the `relativeImport` on Windows has backslashes "\\" in the path instead of forward slashes "/" which is why this relative proxy import plugin breaks on Windows. The fix is straightforward which is to normalize the relativeImport path.

[snowpack-plugin-relative-proxy-imports.js](https://github.com/karolis-sh/electron-snowpack/blob/main/packages/electron-snowpack/lib/snowpack-plugin-relative-proxy-imports.js)
```js
...
if (firstLine === `export default "/${relativeImport}";`) {
...
```

Secondly, the when the bundle/minify flag under the Snowpack optimize config are enabled it breaks as well. I assume under bundling it breaks because no *.proxy.js files are emitted. And for minification it changes `export default "/someimport"` to `export default"/someimport"` and the if statement doesn't get checked.

I suppose the fix for minification, you can remove check for the import without the space. Not exactly sure what you would do for bundling one 

Given how brittle this plugin is, I'm wondering if there's a better way to alter the export.

**Comment by karolis-sh** [comment]
Yeah, regards the minification, it's a bit of an issue (https://github.com/karolis-sh/electron-snowpack/issues/23)

**Comment by water-a** [comment]
Wasn't aware of that #23 issue! I guess the Windows thing is new however!

**Comment by karolis-sh** [comment]
Should be fixed in https://github.com/karolis-sh/electron-snowpack/releases/tag/electron-snowpack%400.12.1

I don't have a Windows machine ready for development, so if the issue persists, let me know ✌️

**Comment by water-a** [comment]
Just got around to my windows machine, and I don't think using `path.join` is correct as it outputs \\.

For example, doing `require('path').join("car", "break", "accident")` results in this:
![image](https://user-images.githubusercontent.com/38114545/145148091-9a59dd46-3f55-4988-925a-95b228303e25.png)

where as we want it to be normalized to use "/"! I'll make a PR.

**Comment by karolis-sh** [comment]
Fix in `electron-snowpack@0.12.2`
